tests/conftest.py

```diff
@@ -411,6 +411,18 @@ def tf_full_py_version(tf_full_version):
     return "py39"
 
 
+@pytest.fixture(scope="module")
+def pytorch_ddp_py_version():
+    return "py3"
+
+
+@pytest.fixture(
+    scope="module", params=["1.10", "1.10.0", "1.10.1", "1.11", "1.11.0", "1.12", "1.12.0"]
```

Comment:
1.10.2 is missing, also is 1.10.1 a valid version that we released? we may have skipped it

Reply:
Gotcha, I will switch to 1.10.2 then.